apps/www/_blog/2023-04-01-supaclub.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: SupaClub
+description: The Worlds First Software Engineering Nightclub.
+author: ant_wilson
+image: supaclub/supaclub-thumb.png
+thumb: supaclub/supaclub-thumb.png
+tags:
+  - announcements
+date: '2023-04-01'
+toc_depth: 3
+---
+
+Announcing SupaClub (pronounced “Super-Clurb”), the world’s first Software Engineering Nightclub.
+
+Located on the Indonesian island of Bali, SupaClub is where the digital nomads of South East Asia are mixing Hardcore Techno with Hardcore Engineering.
+
+![Main Dance Floor](/images/blog/supaclub/supaclub1.png)
+
+In March this year, some members of the Supabase team took advantage of the fact that they can work fully remotely, and spent a week working in Bali. The island is a great place to relax in coffee shops during the day, but when night time rolled around our team faced a major problem - the large established nightclubs of Bali would not allow our engineers to bring their laptops into the club. They have strict rules against EMACS on the dance floor, killing our dream of fixing bugs whilst jamming to 90s club anthems.
+
+Our team got together and decided that we would do what we do best at Supabase. We’d ship a fix, and today that fix becomes a reality.
+
+![coding in the club](/images/blog/supaclub/supaclub2.png)
+
+SupaClub, situated near the beach in the Canggu neighborhood of Bali, will open its doors this coming Friday. It can house 4000 developers across 4 floors, each with a dedicated DJ and breakout rooms. It has UltraSpeed 1Gbps WiFi and bookable standing desks on every floor. VIP [SupaBrew bottle service](https://supabase.com/blog/supabrew) is available inside private conference rooms, complete with whiteboards and UltraWide monitors for full immersion (Think WeWork meets Pacha).
```

Comment:
```suggestion
SupaClub, situated near the beach in the Canggu neighborhood of Bali, will open its doors this coming Friday. It can house 4000 developers across 4 floors, each with a dedicated DJ and breakout rooms. It has UltraSpeed 1Gbps WiFi and bookable standing desks on every floor. VIP [SupaBrew bottle service](https://supabase.com/blog/supabrew) is available inside private conference rooms, complete with whiteboards and UltraWide monitors for full immersion (think WeWork-meets-Pacha).
```